diff --git a/workbench_scheduler.module b/workbench_scheduler.module
index cbee231..c90ea33 100644
--- a/workbench_scheduler.module
+++ b/workbench_scheduler.module
@@ -98,7 +98,9 @@ function workbench_scheduler_manage_schedules_access_check() {
     // Get list of node types that can have schedules.
     $types = workbench_scheduler_get_types();
     // Check if this node type is in the types array.
-    return in_array($node->type, $types);
+    if (isset($types)) {
+      return in_array($node->type, $types);
+    }
   }
 
   return FALSE;
